tests/app/dao/test_services_dao.py

```diff
@@ -380,6 +384,44 @@ def test_get_all_user_services_should_return_empty_list_if_no_services_for_user(
     assert len(dao_fetch_all_services_by_user(user.id)) == 0
 
 
+@freeze_time('2019-04-23T10:00:00')
```

Comment:
We could have a test to ensure the date filter is working.

Reply:
one of ft_biling instances in the test is dated at 2018 and does not appear in the results, so it kind of is tested, or do you mean like a separate test?

Reply:
I added comments to this test so it's clearer what happens there :)